localstack/services/dynamodbstreams/dynamodbstreams_api.py

```diff
@@ -50,13 +52,13 @@ def add_dynamodb_stream(
         )
 
 
-def get_stream_for_table(table_arn):
+def get_stream_for_table(table_arn: str) -> dict:
     region = DynamoDBStreamsBackend.get()
     table_name = table_name_from_stream_arn(table_arn)
     return region.ddb_streams.get(table_name)
 
 
-def forward_events(records):
+def forward_events(records: dict) -> None:
```

Comment:
nit-pick: we tend to use generic types in annotation, i.e., `typing.Dict` in this specific case.